docs/sources/release-notes/v3-2.md

```diff
@@ -73,6 +74,15 @@ Out of an abundance of caution, we advise that users with Loki or Grafana Enterp
 
 ## Bug fixes
 
+### 3.2.2 (2024-12-04)
+
+- **ci:** Fix the LogQL Analyzer CI ([#14511](https://github.com/grafana/loki/issues/14511)).
```

Comment:
did release-please include the `ci` commit? We probably don't need these in release notes.

Reply:
I wasn't sure.  I can remove it.